docs/de/programming/python/tutorial-1.md

```diff
@@ -0,0 +1,130 @@
+---
+nav-title: The First App
+nav-pos: 1
+---
+# Programmieren in Python: die erste Application
+
+Dieses Tutorial soll den Einstieg erleichtern und die Grundschritte erklären!
+
+Eine Application besteht aus **3** Teilen:
+
+ * Das **Programm**. Dies ist normalerweise ein Python-Script. Es könnte aber auch jede beliebige andere Programmiersprache sein Seit [Python](https://www.python.org) der Standart für die TXT-Programmierung ist wird dieses Tutorial sich ausschließlich darauf beziehen!
+
+ * Ein **Manifest**. Das ist eine kleine Datei die Den Programmnamen, Autor, ... angibt
+
+ * Ein **Icon**. Dieses sollte ein PNG-Bild sein mit 64x64 Pixeln. Es wird sowohl auf dem Betriebssystem als auch auf der Weboberfläche verwendet!
+
+## Das Programm
+
+TDas Programm kann irgendeine ausfühbare (_vom TXT_) Datei sein. Solange das Programm vom Launcher (_Startbildschirm_) ausgeführt wird erwartet der Benutzer eine Anzeige des Programms. Deshalb sollte das Programm zumindestens eine minimalistische Oberfläche enthalten.
+
+Aktuell benützen alle Applicationen das [Qt4-Framework](http://www.qt.io/) für ihre Anzeigen. 
+Eine minimale Anwendung kuckt so aus:
+```
+#! /usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import sys
+from TouchStyle import *
+
+class FtcGuiApplication(TouchApplication):
+    def __init__(self, args):
+        TouchApplication.__init__(self, args)
+
+        # Erstelle eine leeres Hauptfenster
+        w = TxtWindow("Test")
+        w.show()
+        self.exec_()        
+
+if __name__ == "__main__":
+    FtcGuiApplication(sys.argv)
+```
+
+Speichere diese Datei unter [`Test.py`](https://raw.githubusercontent.com/ftCommunity/ftcommunity-apps/master/packages/app_tutorial_1/test.py)
+
+**Erkärung:**
+Dieses Programm refenreziert die Klasse TouchApplication, welche die Fenster verwaltet. Jene erstellt ein Fenster und stellt sie dem Nutzer bereit, solange die Application nicht geschlossen wird!
```

Comment:
Der Satz ist etwas sehr holprig, und "referenziert" ist auch nicht der richtige Begriff, um "This app subclasses the TxtApplication class" zu übersetzen.

Besser: "Dieses Programm definiert eine von `TouchApplication` (importiert aus dem Paket `TouchStyle`) abgeleitet Klasse, die ein Fenster mit dem Titel "Test" erstellt."